src/reactions/ReactionIndicator.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2024 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only
+Please see LICENSE in the repository root for full details.
+*/
+
+import { PropsWithChildren, ReactNode } from "react";
+import classNames from "classnames";
+import "@formatjs/intl-durationformat/polyfill";
```

Comment:
This import seems out of place since Intl.DurationFormat is only used in the other file